### PR TITLE
Add PagoEfectivo

### PR DESCRIPTION
**Pull request by joelibaceta**
**Checklist**

- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by joelibaceta** [comment]
Thanks @tauthomas01, I made the fixes

**Comment by joelibaceta** [comment]
Thanks @tauthomas01 don't worry :)

**Comment by joelibaceta** [comment]
@tauthomas01 solved :D

**Comment by joelibaceta** [comment]
@tauthomas01 solved, but I got a non related error in ruby 2.4.4

**Comment by tauthomas01** [comment]
@joelibaceta We're having some issues regarding the build with ruby 2.4.4 so I'll let you know once it's fixed.

**Comment by tauthomas01** [comment]
@joelibaceta You can now rebase your branch off `master` branch to get the latest changes.
Thank you for waiting.

**Comment by joelibaceta** [comment]
@tauthomas01 Done :)

File: app/assets/images/payment_icons/pagoefectivo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...1-4.719-15.871-4.612z" fill="#f4c948"/><path d="M28.8 10.382l-.429-.429-1.5-.751c-1.609 6.542-12.225 11.367-12.225 11.367C11.322 18.424 11 9.094 11 4.59V2.875a3.841 3.841 0 00-.429 1.18v.429a48.629 48.629 0 00-.965 7.936 39.792 39.792 0 00.643 7.185 2.163 2.163 0 002.359 1.93 3.855 3.855 0 001.18-.214 54.116 54.116 0 006.863-2.788 49.429 49.429 0 007.721-4.933 1.952 1.952 0 00.643-2.788.813.813 0 00-.215-.43z" fill="#f4c948"/><path d="M27.831 9.631l-.965-.751a53.671 53.671 0 00-3-2.145 44.751 44.751 0 00-9.652-4.719 2.563 2.563 0 00-3.217.858V4.59s3.324-2.252 15.872 4.611l1.394.751z" fill="#f4c948"/><path d="M17.428 13.063l2.359-1.93.858-.751c.107-.107.214-.214 0-.322l-2.359-2.788a2.02 2.02 0 00-.214.858.284.284 0 01-.322.322h-3.324c-.214 0-.322 0-.214-.322a6.834 6.834 0 00.214-.751.692.692 0 01.643-.643 21.537 21.537 0 015.255 0 4.37 4.37 0 011.93.536 2.46 2.46 0 011.072 2.574c-.107.429-.322.858-.429 1.394a3.8 3.8 0 01-3.324 2.574 13.054 13.054 0 01-3.539.107c-.214 0-.322 0-.429.322a28.733 28.733 0 01-.751 2.9.473.473 0 01-.643.429h-1.609c-.429 0-.536-.107-.429-.429l1.287-5.148c0-.214.107-.214.322-.214h3.539c.214 0 .322 0 .214.322a2.969 2.969 0 00-.107.965z" fill="#303030"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/pagoefectivo.svg]: I just noticed the icon needs a white background and a border. Could you add these?

Right now, the background is transparent: [alt](https://screenshot.click/18-32-3vt9v-lb8q4.png)

In the [contribution](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#appearance) document, you can add this code to your SVG icon :
```
<path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/>
<path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/>
```

Comment (joelibaceta) [reply app/assets/images/payment_icons/pagoefectivo.svg]: Sorry I lost the changes in the last action, I'm going to restore it

**Pull request merged**